ci-operator/step-registry/baremetal/lab/upi/install/baremetal-lab-upi-install-chain.yaml

```diff
@@ -2,8 +2,8 @@ chain:
   as: baremetal-lab-upi-install
   steps:
     - ref: baremetal-lab-upi-install
+    - ref: enable-qe-catalogsource-disconnected
```

Comment:
```suggestion
    - ref: enable-qe-catalogsource-disconnected
    - ref: mirror-images-tag-images
    - ref: set-sample-operator-disconnected
```

Reply:
We need a separate chain `baremetal-lab-upi-install-disconnected`. I will add it with these suggestions.